examples/DotNetCore/AzureFunction/FunctionApp/ReadQueuedMessage.cs

```diff
@@ -0,0 +1,17 @@
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+
+namespace FunctionApp
+{
+    public static class ReadQueuedMessage
+    {
+        // Queue triggered function with queue name defined in Azure App Configuration
+        [FunctionName("ReadQueuedMessage")]
+        public static void Run(
+            [QueueTrigger(queueName: "%TestApp:Storage:QueueName%")] string myQueueItem, 
```

Comment:
I can't find any documentation on the `%settingName%` syntax. Were you able to? I'd like to have a link to it because at first it's a bit hard to understand what's happening here.

Reply:
Here is the doc. It's the binding expression of app settings. Do you think I should include the link in the comment?
https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-expressions-patterns#binding-expressions---app-settings

Reply:
I think adding the link in a comment would be great.

Reply:
Updated. Good suggestion. Thanks!